clarity/src/vm/analysis/arithmetic_checker/tests.rs

```diff
@@ -44,10 +43,11 @@ fn check_good(contract: &str) {
     ArithmeticOnlyChecker::run(&analysis).expect("Should pass arithmetic checks");
 }
 
-#[test]
-fn test_boot_definitions() {
-    check_good(BOOT_CODE_COSTS);
-}
+// #[test]
```

Comment:
:eyes:

Reply:
Moved to boot::contract_tests in https://github.com/stacks-network/stacks-blockchain/commit/9e1f96ad419cd6a620c52f4bc040c144fd71e826